$MirOS: ports/www/firesomething/patches/patch-browser_branding_unofficial_pref_firefox-branding_js,v 1.1.2.1 2008/12/04 21:50:56 bsiegert Exp $
--- browser/branding/unofficial/pref/firefox-branding.js.orig	Tue Dec  2 21:19:30 2008
+++ browser/branding/unofficial/pref/firefox-branding.js	Tue Dec  2 21:19:53 2008
@@ -1,5 +1,3 @@
-pref("startup.homepage_override_url","http://www.mozilla.org/projects/%APP%/%VERSION%/whatsnew/");
-pref("startup.homepage_welcome_url","http://www.mozilla.org/projects/%APP%/%VERSION%/firstrun/");
 // URL user can browse to manually if for some reason all update installation
 // attempts fail.
 pref("app.update.url.manual", "http://www.mozilla.org/products/%APP%/");
